woocommerce/woocommerce
# pin php to <8.3 for package compatitbility

### Changes proposed in this Pull Request:





The `league/commonmark` package has dependencies that aren't compatible with PHP 8.3. This is causing the `Highlight temmplate changes`. CI workflow to fail. Pinning the `woocommerce-docs` plugin to PHP to < 8.3 for now unit we decide what to do with the plugin.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check CI run
2.
3.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
